packages/builder/src/stores/portal/apps.ts

```diff
@@ -146,22 +174,23 @@ export class AppsStore extends BudiStore {
 export const appsStore = new AppsStore()
 
 export const sortBy = derived([appsStore, auth], ([$store, $auth]) => {
-  return $store.sortBy || $auth.user?.appSort || "name"
+  return $store.sortBy || ($auth.user as User | null)?.appSort || "name"
 })
 
 // Centralise any logic that enriches the apps list
 export const enrichedApps = derived(
   [appsStore, auth, sortBy],
   ([$store, $auth, $sortBy]) => {
-    const enrichedApps = $store.apps
-      ? $store.apps.map(app => ({
-          ...app,
-          deployed: app.status === AppStatus.DEPLOYED,
-          lockedYou: app.lockedBy && app.lockedBy.email === $auth.user?.email,
-          lockedOther: app.lockedBy && app.lockedBy.email !== $auth.user?.email,
-          favourite: $auth.user?.appFavourites?.includes(app.appId),
-        }))
-      : []
+    const enrichedApps: EnrichedApp[] = $store.apps.map(app => {
+      const user = $auth.user as User | null
+      return {
+        ...app,
+        deployed: app.status === AppStatus.DEPLOYED,
+        lockedYou: app.lockedBy != null && app.lockedBy.email === user?.email,
```

Comment:
Why is this a simple `!= null` check instead of using !!app.lockedBy

Reply:
No reason! It's an object so both are functionally equivalent - I just tend to use `!= null` more often for some reason.